runatlantis.io/docs/terraform-versions.md

```diff
@@ -15,13 +15,17 @@ projects:
 See [atlantis.yaml Use Cases](repo-level-atlantis-yaml.html#terraform-versions) for more details.
 
 ## Via terraform config
-Alternatively, one can use the terraform configuration block's `required_version` key to specify an *exact* version:
+Alternatively, one can use the terraform configuration block's `required_version` key to [specify an exact version or version constraint](https://developer.hashicorp.com/terraform/language/expressions/version-constraints#version-constraint-syntax):
```

Comment:
It would be good to make a note of the `v0.21.0` release as well.

Before the `v0.21.0` release, only `"x.y.z"` and `"= x.y.z"` would download the appropriate terraform version.

Now `~>` and `=>` also can be used.